library/src/scala/tasty/Reflection.scala

```diff
@@ -2679,14 +2679,30 @@ trait Reflection { reflection =>
       /** Annotations attached to this symbol */
       def annots: List[Term]
 
+      /** Does this symbol come from a currently compiled source file? */
       def isDefinedInCurrentRun: Boolean
```

Comment:
This can be potentially ambiguous, as now the compiler may compile a batch of files in multiple runs to support macros.